setup-dev-env.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+
+echo "Updating python and installing ansible modules"
```

Comment:
FYI: It would look nice if you colorize the console outputs.

```suggestion
echo "\e[36mUpdating python and installing ansible modules\e[m"
```